jmxtrans-output/jmxtrans-output-core/pom.xml

```diff
@@ -53,6 +53,11 @@
 			<artifactId>commons-jexl</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.kafka</groupId>
```

Comment:
The goal of the `jmxtrans-output-core` module is to have only core output writers, with minimal dependencies. Could you move the output writer to a new module, probably named `jmxtrans-output-kafka` ?